epoch/api/README.md

```diff
@@ -62,6 +63,33 @@ A response (and an event) has the format:
 | action | string | what was the action | Yes |
 | payload | object | data from the action | Yes |
 
+## Channels WebSocket API definition
+
+### Description
+Channels require persisted connections to an Aeternity node. New state is
+co-signed by both parties and then it becomes the latest valid state of the
+channel. This happens off chain. For persistence of the connection websockets
+are used.
+
+### Connection
+The epoch node supports an endpoint with a configurable port where the
+websocket's clients connect. It is located on `/channel`.
+
+The node could serve multiple channel websocket clients. Their number is configured in
+the epoch.yaml. When all websocket connections are consumed - any new incoming
+connections will be queued. The queue has a maximum size and when it is
+reached - any new incoming connections will be rejected with an error code 400.
+This is to prevent the node of being overloaded with websocket connections.
+
+### General message types
+All messages have the format:
+
+| Name | Type | Description | Required |
+| ---- | ---- | ----------- | -------- |
+| action | string | what is the action | Yes |
+| tag | string | additional info for the action | No |
```

Comment:
Please rebase on master - that has related improvement https://github.com/aeternity/protocol/commit/cdf2d8eb5560d2d6339befcd7cad096502697162

Reply:
Rebased on master but the two different WS endpoints don't share much...